ERCS/erc-7688.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 7688
```

Comment:
```suggestion
eip: 7606
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.